spec/models/customer_spec.rb

```diff
@@ -118,7 +118,7 @@
       let(:organization_invoice_grace_period) { 5 }
 
       before do
-        customer.invoice_grace_period = 0
+        customer.invoice_grace_period = nil
```

Comment:
if it's null by default, no need to initialize it?

Reply:
nope, here `invoice_grace_period` is equal to `3`, see [here](https://github.com/getlago/lago-api/pull/687/files/090c606ccc60595fe1cb2cf146d97035d76ca223#diff-169a4b470e704cdc458507df849294eaa7005d1db666def55bb70f25f5d3a02cL110)